src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -1497,7 +1499,12 @@ well as menu structures (for main menu and popup menus).
         label="Show All Panes"
         menuLabel="_Show All Panes"
         windowMode="main"/>
-        
+
+   <cmd id="newSourceColumn"
```

Comment:
For easier discovery, recommend adding the command to the (increasingly crowded) View menu, probably under the Pane submenu.

Reply:
It is added to the View menu unless I'm misunderstanding:

![image](https://user-images.githubusercontent.com/5323711/88116937-a8612000-cb6e-11ea-8e1d-b278a397f4cd.png)

Reply:
Oops, my bad, sorry. Missed that.